amp-script-tutorial/README.md

```diff
@@ -0,0 +1,29 @@
+# amp-script-tutorial
+
+## About
+
+This repository contains the code you'll use to do the [&lt;amp-script&gt; tutorial](https://amp.dev/documentation/guides-and-tutorials/develop/custom-javascript-tutorial/)!
```

Comment:
I'd say the name of the tutorial, not just "the amp-script" tutorial. We may have more?

Reply:
I doubt that we will have more. But... yes, I see your point.

I was calling this "Use JavaScript in AMP". How would you phrase this, then? I'd normally say something like, "the &lt;amp-script&gt; tutorial: 'Use JavaScript in AMP.'" Would you say, instead, "the 'Use JavaScript in AMP' tutorial'?

Reply:
Maybe "This repository contains the code you need to follow the Build custom UI with amp-script tutorial"

Reply:
ok! (though I have also changed the title)